src/css/collapsible.css

```diff
@@ -27,7 +27,9 @@ h2 {
 .Collapsible__contentInner p {
   margin-bottom: 10px;
   font-size: 14px;
-  line-height: 20px; 
+  line-height: 20px;
+  display: flex;
+  justify-content: space-between;
```

Comment:
This affects the display of the Locations to Compare section too. I'd like to see a more specific CSS selector for these lines and a CSS class specific to groups of metrics, like `className={comparisonFts.length ? "comparison" : ""}` on line 18 of LocationDetails.js and then break these 3 lines into `.Collapsible__contentInner .comparison>p { ... }`

Reply:
Not really, because in Locations to Compare section they used a `<div>` and not a `<p>` so it's not affected. 
But I'll add a class to `<p>` when is comparing.

Reply:
The "Add up to 4 regions." text uses a p tag. Its center alignment was getting overridden to left alignment.